ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/map.view.js

```diff
@@ -197,10 +198,13 @@ module.exports = Marionette.LayoutView.extend({
         this.map.zoomToBoundingBox(homeBoundingBox !== undefined ? homeBoundingBox : defaultHomeBoundingBox);
     },
     addHome: function() {
-        this.$el.find('.cesium-viewer-toolbar').append('<button class="is-button zoomToHome"><span class="fa fa-home"></span><span> Home</span></div>');
+        this.$el.find('.cesium-viewer-toolbar').append('<div class="is-button zoomToHome">' +
```

Comment:
✏️ It would be nice to actually refactor this out into its own view someday so that we don't need to do this

Reply:
Maybe a ticket would be nice?